### DISCUSSION TITLE
Adjust message if SENDS_ALLOWED=false

### DISCUSSION THREAD

**Discussion by Tealk** [dani-garcia/vaultwarden#5059] [Ideas]
If I set the following setting to false:

https://github.com/dani-garcia/vaultwarden/blob/f0efec7c9623fab4ba3c2b53cba724e4e6a8a739/.env.template#L190

the following message is displayed when sending:

> Error
> 
> An error has occurred.
> 
> Due to a company policy, it is only possible to delete an existing shipment.

Would it be possible to remove the error and just point out that the function is deactivated?

**Comment by stefan0xC** [comment]
It is possible if we check for it separately:
https://github.com/dani-garcia/vaultwarden/blob/f0efec7c9623fab4ba3c2b53cba724e4e6a8a739/src/api/core/sends.rs#L83-L87

**Reply by Tealk** [reply]
I'm not so good at the language, but I think you have to pass it off as info and not as an error?

**Reply by stefan0xC** [reply]
Nope. You have to check for it separately, e.g. 
```rust
if !CONFIG.sends_allowed() 
{ 
    err!("Sends have been disabled.") // or whatever your suggestion would be
} 
if OrgPolicy::is_applicable_to_user(user_uuid, OrgPolicyType::DisableSend, None, conn).await 
{ 
    err!("Due to an Enterprise Policy, you are only able to delete an existing Send.") 
} 
```
